subsys/usb/common/include/usb_uvc.h

```diff
@@ -435,7 +453,7 @@ struct uvc_frame_discrete_descriptor {
 	uint32_t dwMaxVideoFrameBufferSize;
 	uint32_t dwDefaultFrameInterval;
 	uint8_t bFrameIntervalType;
-	uint32_t dwFrameInterval[CONFIG_USBD_VIDEO_MAX_FRMIVAL];
+	uint32_t dwFrameInterval[CONFIG_USBH_VIDEO_MAX_FRMIVAL];
```

Comment:
This is being used by both USB and USB device, so a Kconfig option that is independent from host/device is needed.

I moved that to a dedicated commit in API3 so that it can be used for both device and host.